chart/fluentbit-operator/templates/fluentbit-operator-deployment.yaml

```diff
@@ -40,13 +40,8 @@ spec:
       - name: fluentbit-operator
         image: {{  .Values.fluentbitOperatorImage.repository   }}:{{  .Values.fluentbitOperatorImage.tag }}
         resources:
-          limits:
-            cpu: 100m
-            memory: 30Mi
-          requests:
-            cpu: 100m
-            memory: 20Mi
+          {{ toYaml .Values.resources.operator | indent 12 }}
```

Comment:
Maybe {{  toYaml .Values.resources.operator.resources | nindent 10  }} is better.